src/TwigRendererFactory.php

```diff
@@ -29,7 +30,10 @@
 class TwigRendererFactory
 {
     /**
-     * @throws Exception\InvalidConfigException for invalid config service values.
+     * @param ContainerInterface $container
+     *
+     * @return TwigRenderer
+     * @throws LoaderError
```

Comment:
Please revert the `@param` and `@return` annotations, and detail when a `LoaderError` is thrown.